### PR TITLE
fix(Table): fixed empty and shadow issues when virtual mode

### PR DESCRIPTION
**Pull request by linxianxi**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [x] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

fix https://github.com/ant-design/ant-design/issues/48991
fix https://github.com/ant-design/ant-design/issues/48959
fix https://github.com/ant-design/ant-design/issues/49185

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     support minWidth, fixed empty and shadow issues when virtual mode      |
| 🇨🇳 Chinese |     支持 minWidth，修复虚拟表格下的空数据和阴影问题      |

### CODE REVIEW

**Comment by linxianxi** [comment]
大多数 snapshot 更新都是由 https://github.com/react-component/table/pull/1110 引起的

**Comment by yoyo837** [comment]
7.46.0 没有新特性，7.47.0 有个 https://github.com/react-component/table/pull/1125，看是否需要加文档或者demo？

**Comment by linxianxi** [comment]
> 7.46.0 没有新特性，7.47.0 有个 [react-component/table#1125](https://github.com/react-component/table/pull/1125) ，看是否需要加文档或者demo？
这个感觉可以先不写，能用的情况太少，tableLayout 大多数都是 fixed。这个 api 主要还是给表头拖拽做限制的

**Comment by yoyo837** [comment]
再同步一次

**Comment by linxianxi** [comment]
> 再同步一次

已经是 feature 最新的了

**Comment by yoyo837** [comment]
@afc163 老板确认image diff没问题我们可就合了哟 :)

**Comment by afc163** [comment]
https://github.com/react-component/table/pull/1125/ 这个要补一下。

File: components/table/__tests__/__snapshots__/empty.test.tsx.snap [hunk 1]
```diff
@@ -395,68 +420,79 @@ exports[`Table renders empty table with fixed columns should work 1`] = `
...
                     class="ant-table-cell"
                     scope="col"
+                    title="Column 2"
                   >
                     Column 2
                   </th>
                   <th
                     class="ant-table-cell"
                     scope="col"
+                    title="Column 3"
```
Comment (afc163) [inline components/table/__tests__/__snapshots__/empty.test.tsx.snap:452]: 这里都会默认加上 title，也蛮干扰视线的，感觉会有用户反馈。

**Comment by afc163** [comment]
https://github.com/react-component/table/pull/1176 里回滚了，需要升级 rc-table 版本并更新一下 snapshot。

File: package.json [hunk 2]
```diff
@@ -138,7 +138,7 @@
     "rc-slider": "~11.1.5",
     "rc-steps": "~6.0.1",
     "rc-switch": "~4.1.0",
-    "rc-table": "~7.45.7",
+    "rc-table": "~7.47.2",
```
Comment (afc163) [inline package.json:141]:
```suggestion
    "rc-table": "~7.47.3",
```

**Comment by yoyo837** [comment]
@SocketSecurity ignore npm/sharp@0.33.4

**Comment by yoyo837** [comment]
之前的 image diff 老板已经 approve 过了，重新跑的我就直接 approve 了哈。

**Review by yoyo837 — APPROVED** [review]

**Pull request merged**